Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -445,7 +451,7 @@ std::map<std::string, std::string> FitPeaks::validateInputs() {
   map<std::string, std::string> issues;
 
   // check that min/max spectra indices make sense - only matters if both are specified
-  if (!(isDefault(PropertyNames::START_WKSP_INDEX) && isDefault(PropertyNames::START_WKSP_INDEX))) {
+  if (!(isDefault(PropertyNames::START_WKSP_INDEX) && isDefault(PropertyNames::STOP_WKSP_INDEX))) {
```

Comment:
This looks like a bug fix in `FitPeaks`? If so, it probably deserves a separate release note

Reply:
This has been fixed in  #37843
I will rebase the PR with main to include the fix